scripts/vscode-run-tests.ps1

```diff
@@ -0,0 +1,56 @@
+param (
+    [Parameter(Mandatory=$true)][string]$filePath,
+    [Parameter(Mandatory=$true)][string]$framework,
+    [Parameter(Mandatory=$false)][string]$filter
+)
+
+Set-StrictMode -version 3.0
+$ErrorActionPreference="Stop"
+
+Push-Location
+
+$fileInfo = Get-ItemProperty $filePath
+Set-Location $fileInfo.Directory
+
+try
+{
+    while ($true)
+    {
+        # search up from the current file for a folder containing a csproj
+        $files = Get-ChildItem *.csproj
+        if ($files)
+        {
+            Pop-Location
+            $dotnetPath = Resolve-Path "$PSScriptRoot/../.dotnet/dotnet.exe" -Relative
```

Comment:
Did u consider importing the build-utils script which has our definitive way of grabbing the dotnet binary?

Reply:
Please do this so I can use this on Linux.